### DISCUSSION TITLE
Where to configure passwordFromEnv

### DISCUSSION THREAD

**Discussion by Savitha-Amaldev** [kedacore/keda-docs#461] [Q&A] [answered]
I'm pretty new to keda. I'm writing a scaledobject with MySQL trigger and I saw "passwordFromEnv" where password needs to be mentioned. Can someone tell me where are we configuring this environment variable specific to keda?

I have a secret.yaml where i configured the password in base64 encoded form and deployed the same. But scaledobject READY and ACTIVE status is still unknown.

**Comment by tomkerkhove** [comment, marked as answer]
You can use `passwordFromEnv` which will read the password from an environment variable on the scale target.

However, we recommend using TriggerAuthentication instead where you have more ways to look up secrets, for example from a Kubernetes secret.

https://keda.sh/docs/2.3/concepts/authentication/#secrets
